src/workerd/jsg/resource.h

```diff
@@ -527,14 +533,15 @@ template <typename TypeWrapper,
     const char* methodName,
     typename Method,
     Method method,
-    bool isContext>
+    bool isContext,
+    bool memoize>
```

Comment:
Non-blocking nit... really not a big fan of bool args like this. Much prefer enums for this type of case. Won't block on it tho